core/src/main/java/apoc/convert/Json.java

```diff
@@ -41,8 +41,9 @@ private Object writeJsonResult(Object value) {
             case MAP:
                 return ((Map<String, Object>) value).entrySet()
                         .stream()
-                        .collect(Collectors.toMap(Map.Entry::getKey,
-                                e -> writeJsonResult(e.getValue())));
+                        .collect(HashMap::new, // workaround for https://bugs.openjdk.java.net/browse/JDK-8148463
```

Comment:
@conker84 you can fix this JDK bug in hack-commit-push with Rémi Forax if you want :P